src/kalastatic.js

```diff
@@ -51,10 +53,16 @@ KalaStatic.prototype.build = function () {
     var base = config.get('base')
     var metalsmith = new Metalsmith(base)
     var source = config.get('source')
+    var destination = config.get('destination')
 
     // Retrieve the Plugin configuration.
     var plugins = config.get('plugins')
     var pluginDefaults = {
+      'metalsmith-define': {
+        // Expose both a base_path and a build_path variables.
+        'base_path': '/', // eslint-disable-line quote-props
```

Comment:
Not really needed.... eslint-disable-line will disable eslint warnings for that line:
http://eslint.org/docs/rules/quote-props